packages/vm/src/modules/file_system_cache.rs

```diff
@@ -310,11 +310,11 @@ mod tests {
         };
         let target = Target::new(triple.clone(), wasmer::CpuFeature::POPCNT.into());
         let id = target_id(&target);
-        assert_eq!(id, "x86_64-nintendo-fuchsia-gnu-coff-4721E3F4");
+        assert_eq!(id, "x86_64-nintendo-fuchsia-gnu-coff-01E9F9FE");
```

Comment:
Any idea why these changed?

Reply:
Honestly, no.
Tried to figure it out by investigating dependency checking, but it gave me nothing...

This is the code, where this strings is formed:

```
/// Creates an identifier for the Wasmer `Target` that is used for
/// cache invalidation. The output is reasonable human friendly to be useable
/// in file path component.
fn target_id(target: &Target) -> String {
    // Use a custom Hasher implementation to avoid randomization.
    let mut deterministic_hasher = crc32fast::Hasher::new();
    target.hash(&mut deterministic_hasher);
    let hash = deterministic_hasher.finalize();
    format!("{}-{:08X}", target.triple(), hash) // print 4 byte hash as 8 hex characters
}
```

..but neither crc32fast, nor Target was changed.

Maybe @webmaster128 can give a clue...